src/makers/cotx/hotspots.ts

```diff
@@ -0,0 +1,62 @@
+import HotspotIcon from './cotx.svg'
+import { MakerHotspot } from '../hotspotMakerTypes'
+import ANTENNAS from './antennas'
+
+const COTX_V1 = {
+  name: 'COTX Gateways',
```

Comment:
You have some lint errors in this file you need to fix. You can see them by running `yarn lint` in the root folder:
```
  30:59  error  Replace `\·n·\` with `·n·`                                                                                                                                                     prettier/prettier
  30:59  error  Unnecessary escape character: \ .                                                                                                                                              no-useless-escape
  30:63  error  Unnecessary escape character: \ .                                                                                                                                              no-useless-escape
  59:13  error  Replace `·us:·ANTENNAS.COTX_US,·default:·ANTENNAS.COTX_EU,·cn:·ANTENNAS.COTX_CN` with `⏎····us:·ANTENNAS.COTX_US,⏎····default:·ANTENNAS.COTX_EU,⏎····cn:·ANTENNAS.COTX_CN,⏎·`  prettier/prettier

✖ 4 problems (4 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.
```

Reply:
> You have some lint errors in this file you need to fix. You can see them by running `yarn lint` in the root folder:
> 
> ```
>   30:59  error  Replace `\·n·\` with `·n·`                                                                                                                                                     prettier/prettier
>   30:59  error  Unnecessary escape character: \ .                                                                                                                                              no-useless-escape
>   30:63  error  Unnecessary escape character: \ .                                                                                                                                              no-useless-escape
>   59:13  error  Replace `·us:·ANTENNAS.COTX_US,·default:·ANTENNAS.COTX_EU,·cn:·ANTENNAS.COTX_CN` with `⏎····us:·ANTENNAS.COTX_US,⏎····default:·ANTENNAS.COTX_EU,⏎····cn:·ANTENNAS.COTX_CN,⏎·`  prettier/prettier
> 
> ✖ 4 problems (4 errors, 0 warnings)
>   2 errors and 0 warnings potentially fixable with the `--fix` option.
> ```

The above questions have been revised.

Reply:
@cotxliang Thanks for the lint fixes there is still a merge conflict to resolve

Reply:
Thank you. I just corrected the question. Do you think it’s okay?